examples/hosts/app-integration/external-data/README.md

```diff
@@ -6,15 +6,68 @@ Please note that the ideas explored here are experimental and under development.
 
 ## Scenario
 
-TBD
+This example demonstrates a scenario in which the "source of truth" of the customer data lives in a service that is external to the Fluid service. Customers can then:
+
+1. Import the external data into a Fluid collaboration session.
+2. Export data from a Fluid collaboration session back to the source of truth.
+3. Sync updates between Fluid and the source of truth in as close to real-time as the scenario allows.
+
+In this case, the Fluid collaboration session serves as a "drafting surface" in which clients collaborate to create a draft of the data and then send the saved data back to the source of truth for long term storage.
 
 ## Strategy overview
 
-TBD
+In order to accomplish the goals above, we have split up the responsibilities into a few different pieces:
+
+**External Data Service**
+
+Many services that would hold the "source of truth" data offer explicit commit style interfaces (e.g. vi REST call or similar) which are not well suited to rapid updates.
+However, they often expose third-party integration via REST APIS for querying and manipulating data, as well as webhooks for watching updates to the data.
+
+This repo contains a service that mocks the external "source of truth" data server. This mock service offers a REST API collection and webhook interfaces in ./src/mock-external-data-service. The API requests served by this "external" service are the following:
+
+1. POST `/register-for-webhook`: Register's the sender's URL (in this case the Customer Service's URL) to receive notifications when the external task-list data changes.
+2. GET `/fetch-tasks`: Fetches the task list from the external data store.
+3. POST `/set-tasks`: Updates external data store with new tasks list (complete override).
+4. POST `/debug-reset-task-list`: Resets the external data to its original contents.
+
+Find the details of the API in the [External Data Service README](./src/mock-external-data-service/README.md)
+
+**Customer Service**
+
+Next we need a customer service that functions as the intermediary between the External Data Service and the Fluid Service. This server is responsible for authenticating to the external service on the customer's behalf. It registers to the External Data Service webhooks and listens for incoming changes. It also acts as a translation layer, tasnlating to and from the External Data Service expected format and the Fluid Service's expected format.
+
+In this example, the Customer Service contains the following endpoints:
+
+1. POST `/register-for-webhook`: Register's the sender's URL (in this case, the Fluid Service's URL) to receive notifications when the external task-list data changes.
```

Comment:
This doesn't sound right -- I don't think the customer's service provides any webhooks to be registered for, but instead it registers for webhooks from both the external service and the fluid service?

Reply:
This describes the current pattern, but it probably isn't what we want. What you describe makes a lot more sense, so we should update the customer pattern to know the Fluid service endpoint, and not require any sort of registration from the Fluid service before beginning to echo incoming updates.